engine/src/main/resources/templates/Inventory/dailyEmailTitle.txt

```diff
@@ -0,0 +1 @@
+Insights Inventory daily summary
```

Comment:
A bit of a nitpick, but I wonder if UX has any preference for `Red Hat Insights` over simply `Insights` when in email titles?  This is the case in the body and the button "Open Inventory in Insights" further down as well.  I don't see a consistent approach from other apps, so perhaps more a question for UX review in future and fine for now as is.